src/TxHistory/ActionsBanner.tsx

```diff
@@ -19,6 +20,12 @@ export const ActionsBanner = () => {
   const strings = useStrings()
   const navigateTo = useNavigateTo()
   const wallet = useSelectedWallet()
+  const {sendActions} = useSend()
+
+  const onSend = () => {
+    navigateTo.send()
+    sendActions.clear()
```

Comment:
why?

Reply:
Ticket requirement. Restart the flow => new transaction.